common/classify.py

```diff
@@ -9,6 +9,8 @@
 from sklearn import linear_model, cross_validation
 from sklearn.utils import shuffle
 from sklearn.externals import joblib
+from memtest import MemTracker
```

Comment:
Nevermind ignore my comment, didn't realize memtest was a user-defined inport. For future reference, can we separate our import statements as follows:
```
Standard Python Libraries
-newline-
Non-standard library packages
-newline-
User-defined files
```

I think this will make it easier for readability purposes.